packages/react/src/filter-sidebar/FilterSidebar.tsx

```diff
@@ -40,6 +40,9 @@ export function FilterSidebar({
 
 	return (
 		<CollapsingSideBar
+			ariaLabel={`Filters${
+				activeFiltersCount ? ` (${activeFiltersCount} active)` : ''
```

Comment:
Is it better to use a comma over the brackets to create the pause?

Reply:
I only wanted a small pause, and it makes the pause consistent for when they read the h2

Reply:
Ahh ok, good to know, thanks.